apps/changelog-generator/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@blocksense/changelog-generator",
+  "version": "0.0.0",
+  "type": "module",
+  "license": "MIT",
+  "description": "An Effect app related with Release notes functionality",
+  "scripts": {
+    "build": "tsup --dts && pnpm copy-package-json",
+    "ts": "yarn node --import tsx",
+    "build:ts": "tsup",
+    "lint": "eslint \"**/{src,test,examples,scripts,dtslint}/**/*.{ts,mjs}\"",
+    "lint-fix": "yarn lint --fix",
+    "test": "vitest run",
+    "coverage": "vitest run --coverage",
```

Comment:
To run this, you will need to install a dependency. It prompts you to do so:
```
 Cannot find dependency '@vitest/coverage-v8'

✔ Do you want to install @vitest/coverage-v8? 
```

Let's install it on this branch, so we can have smooth dev experience later